service/src/main/java/com/codeforcommunity/requester/MapRequester.java

```diff
@@ -242,6 +244,16 @@ public void updateBlocks(List<String> blockIds, BlockStatus updateTo) {
         });
   }
 
+  public Object getPrivateStreets() {
+    try {
+      return new JSONParser()
+          .parse(new FileReader("service/src/main/resources/private_streets_light.geojson"));
+    } catch (Exception e) {
+      logger.error("There was an error loading GeoJSON data", e);
+      return new Object();
```

Comment:
Why do we want this to return an object here? Where/how will the object be used? What type does JSONParser.parse() return?

Reply:
I'm not exactly sure how this object will be used. An assumption I had is that some method will call getPrivateStreets() and use its return object for some operations. 

Continuing on that, I changed this method to return a java vertx JsonObject rather than an Object from the jOOQ jsonParser. It seems that other methods in this class use vertx's son library so I wanted to stick to that. So, now it reads in the file's bytes, converts them into a string, and creates a json object based on that.